docs/data/charts/getting-started/getting-started.md

```diff
@@ -78,7 +78,7 @@ pnpm add @mui/styled-engine-sc styled-components
 
 </codeblock>
 
-Take a look at the [Styled engine guide](/material-ui/guides/styled-components/) for more information about how to configure `styled-components` as the style engine.
+Take a look at the [Styled engine guide](/material-ui/integrations/styled-components/) for more information about how to configure `styled-components` as the style engine.
```

Comment:
I assume it's the `docs:link-check` that asked for that 😍

Reply:
Yes, I could've added them to `.link-check-errors` because most of them have redirects, but decided to update the links instead.